src/main/java/com/slack/kaldb/logstore/LuceneIndexStoreImpl.java

```diff
@@ -239,21 +242,21 @@ public String toString() {
   }
 
   @Override
-  public Collection<String> activeFiles() {
-    if (indexWriter.isEmpty()) {
-      throw new IllegalStateException(
-          "This function shouldn't be called when index writer is null");
+  public IndexCommit getIndexCommit() {
+    try {
+      return snapshotDeletionPolicy.snapshot();
+    } catch (IOException e) {
+      LOG.error("Tried to snapshot index commit but failed", e);
     }
+    return null;
+  }
 
-    DirectoryReader directoryReader;
+  public void decIndexCommitRef(IndexCommit indexCommit) {
```

Comment:
Let's rename the function releaseIndexCommitRef to better reflect it's intended function.